openfl-tutorials/experimental/workflow/405_MNIST_FederatedEvaluation.ipynb

```diff
@@ -0,0 +1,471 @@
+    "- `Runtime` – Defines where the flow runs, infrastructure for task transitions (how information gets sent). The `LocalRuntime` runs the flow on a single node.\n",
+    "- `aggregator/collaborator` - placement decorators that define where the task will be assigned"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8e406db6",
+   "metadata": {
+    "scrolled": true
+   },
+   "source": [
+    "Now we come to the flow definition. The OpenFL Workflow Interface adopts the conventions set by Metaflow, that every workflow begins with `start` and concludes with the `end` task. The aggregator begins with an optionally passed in model and optimizer. The aggregator begins the flow with the `start` task, where the list of collaborators is extracted from the runtime (`self.collaborators = self.runtime.collaborators`) and is then used as the list of participants to run the task listed in `self.next`, `aggregated_model_validation`. The model, optimizer, and anything that is not explicitly excluded from the next function will be passed from the `start` function on the aggregator to the `aggregated_model_validation` task on the collaborator. Where the tasks run is determined by the placement decorator that precedes each task definition (`@aggregator` or `@collaborator`). Once each of the collaborators (defined in the runtime) complete the `aggregated_model_validation` task, they finally `join` at the aggregator doing just model evaluation/validation without any training. It is in `join` that an accuracy of model per collaborator is shown.\n",
+    "\n",
+    "![FedEval.png](../../../docs/images/FedEval.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "difficult-madrid",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from copy import deepcopy\n",
+    "\n",
+    "from openfl.experimental.workflow.interface import FLSpec, Aggregator, Collaborator\n",
+    "from openfl.experimental.workflow.runtime import LocalRuntime\n",
+    "from openfl.experimental.workflow.placement import aggregator, collaborator\n",
+    "class FederatedEvaluationFlow(FLSpec):\n",
+    "\n",
+    "    def __init__(self, model=None, rounds=1, **kwargs):\n",
+    "        super().__init__(**kwargs)\n",
+    "        if model is not None:\n",
+    "            self.model = model\n",
+    "        else:\n",
+    "            self.model = Net()\n",
+    "            \n",
+    "        self.rounds = rounds\n",
+    "\n",
+    "    @aggregator\n",
+    "    def start(self):\n",
+    "        print(f'Performing initialization for model')\n",
+    "        self.collaborators = self.runtime.collaborators\n",
+    "        self.private = 10\n",
+    "        self.current_round = 0\n",
+    "        self.next(self.aggregated_model_validation, foreach='collaborators', exclude=['private'])\n",
+    "\n",
+    "    @collaborator\n",
+    "    def aggregated_model_validation(self):\n",
```

Comment:
Could we simply call this step `evaluate()` ?

PS: if you agree, please also change it in the diagram